### PR TITLE
Notify: add zIndex to stack container (T1105343)

### PR DESCRIPTION
**Pull request by Zedwag**

### CODE REVIEW

File: testing/tests/DevExpress.ui/notify.tests.js [hunk 1]
```diff
@@ -171,8 +173,18 @@ QUnit.module('notify', {
...
             const stackStyles = $(`.${TOAST_STACK}`)[0].style;
 
-            assert.equal(stackStyles.bottom, `${position.bottom}px`);
-            assert.equal(stackStyles.right, `${position.right}px`);
+            assert.strictEqual(stackStyles.bottom, `${position.bottom}px`);
+            assert.strictEqual(stackStyles.right, `${position.right}px`);
+            this.clock.tick(100);
+        });
+
+        QUnit.test('stack container should have toasts base zIndex', function(assert) {
```
Comment (ksercs) [inline testing/tests/DevExpress.ui/notify.tests.js]: let's add a ticket id

File: testing/tests/DevExpress.ui/notify.tests.js [hunk 2]
```diff
@@ -171,8 +173,18 @@ QUnit.module('notify', {
 
             const stackStyles = $(`.${TOAST_STACK}`)[0].style;
 
-            assert.equal(stackStyles.bottom, `${position.bottom}px`);
-            assert.equal(stackStyles.right, `${position.right}px`);
+            assert.strictEqual(stackStyles.bottom, `${position.bottom}px`);
+            assert.strictEqual(stackStyles.right, `${position.right}px`);
+            this.clock.tick(100);
```
Comment (ksercs) [inline testing/tests/DevExpress.ui/notify.tests.js:178]: why do we need this tick?

Comment (ksercs) [inline testing/tests/DevExpress.ui/notify.tests.js]: why do we need this tick?
If we really need it it's a good idea to move it to `afterEach`

File: testing/tests/DevExpress.ui/notify.tests.js (hunk 1 above)
Comment (Zedwag) [reply testing/tests/DevExpress.ui/notify.tests.js]: done

File: testing/tests/DevExpress.ui/notify.tests.js (hunk 2 above)
Comment (Zedwag) [reply testing/tests/DevExpress.ui/notify.tests.js]: because of displayTime: 100
moved to aferEach

**Review by ksercs — APPROVED** [review]

**Pull request merged**